### PR TITLE
add boilerplate

### LINKED ISSUE #2: Should we add boilerplate?
Do we care about a candidate showing that they know how to use create-react-app? I would guess no, so I reckon we should add the boilerplate so they can get right into the juicy stuff.

I say we provide an empty app using our react framework and our css-in-js framework. That way we're advertising that we use nice new tech as well!

### PR DESCRIPTION
**Pull request by remyoudemans**
Resolves #2

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { createGlobalStyle, ThemeProvider } from 'styled-components'
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
```
Comment (donofkarma) [inline pages/_app.js:8]:
```suggestion
  * {
    box-sizing: border-box;
  }

  body {
    margin: 0;
    padding: 0;
  }
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "name": "oodle-front-end-technical-test",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "latest",
+    "react": "^16.8.0",
```
Comment (donofkarma) [inline package.json:11]: Let's go for the latest versions.

File: README.md [hunk 3]
```diff
@@ -22,6 +22,7 @@ Note that the designs are for two separate pages: a beer listing page and a page
 - Fetching and Handling Data Asynchronously
 
 ### More specifics
+- clone this repo, it has some boilerplate to help you get started
```
Comment (donofkarma) [inline README.md:25]: If we are asking them to clone should we also be expecting them to submit a PR as their test submission?

If not then forking might be better option.

Comment (remyoudemans) [reply README.md:25]:
```suggestion
- fork this repo, it has some boilerplate to help you get started
```

**Review by donofkarma — APPROVED** [review]

**Pull request merged**